app/controllers/api/v4/runs_controller.rb

```diff
@@ -14,11 +14,30 @@ class Api::V4::RunsController < Api::V4::ApplicationController
   end
 
   def show
-    if params[:historic] == '1'
-      @run.parse(fast: false)
-      render json: @run, serializer: Api::V4::RunWithHistorySerializer
+    accept = request.headers.fetch('HTTP_ACCEPT', 'json').downcase
+    if accept == "original_timer"
+      accept = @run.program
+    end
+    timer = Run.program(accept)
```

Comment:
It looks like we're specifying our outgoing Content-Types as `application/<format>` but expecting consumers to pass us `<format>`; imo we should be using the former for both cases.

This might be a good place to add a new method like `content_type` to each of the parsers, like we do for `file_extension` and `website`, so each parser can be in charge of how its name gets translated into a Content-Type header.

Reply:
So have a class method that just returns something like "application/wsplit", correct?

Reply:
Yeah exactly.

Reply:
For case formatting should it be all down case or with the custom casing like LiveSplit?

Reply:
There's probably some RFC somewhere that has a standard in it. From what I've seen it's all lowercase with hyphens for spaces. So something like `application/livesplit` and `application/time-split-tracker` etc.

Reply:
Gotcha, I'll look around for some kind of specification when I have some time.